src/content/api/stats.mdx

```diff
@@ -176,7 +177,7 @@ The `errors` and `warnings` properties each contain a list of strings. Each stri
 ``` bash
 ../cases/parsing/browserify/index.js
 Critical dependencies:
-2:114-121 This seem to be a pre-built javascript file. Even while this is possible, it's not recommended. Try to require to original source to get better results.
+2:114-121 This seem to be a pre-built javascript file. Even while this is possible, it's not recommended. Try to require the original source to get better results.
```

Comment:
this looks like output from other software, please revert if so

Reply:
@EugeneHlushko to two times was lookin awkward that's why i did this. What u say?

Reply:
@EugeneHlushko just reverted the output to it's original state